src/repositories/PointofSaleRepository.js

```diff
@@ -6,9 +6,24 @@ export const PointofSaleRepository = {
     get: service + 'getUSR',
     set: service + 'setUSR',
     del: service + 'delUSR',
-    page: service + 'pageUSR'
+    page: service + 'pageUSR',
+    set2: service + 'set2USR'
   },
   PosUsersPOS: {
     qry: service + 'qryPOS'
+  },
+  PointOfSales: {
```

Comment:
merge all endpoints you add to the above block directly, already POS endpoint exists